projects/cashmere/src/lib/date-range/picker-overlay/picker-overlay.component.html

```diff
@@ -32,7 +32,7 @@
     <div class="hc-date-range-calendar-item">
         <div class="hc-date-range-menu">
             <hc-radio-group class="presets" [(ngModel)]="_selectedPreset">
-                <hc-radio-button *ngFor="let p of options.presets" [value]="p.range" (change)="_updateRangeByPreset(p.range)">{{ p.presetLabel }}</hc-radio-button>
+                <hc-radio-button *ngFor="let p of options.presets; let i = index" [value]="p.range" [id]="i" (change)="_updateRangeByPreset(p.range)">{{ p.presetLabel }}</hc-radio-button>
```

Comment:
This is a rather minor suggestion, but what do you think about adding a prefix or suffix to the assigned id? As is, it seems like these ids have a somewhat high risk of violating the id uniqueness constraint. Perhaps something like `[id]="'hc-date-range-preset-' + i"`

Reply:
Great suggestion!  Thanks for the review Nick - just pushed a commit with your suggestion.  @jacquie - how does this all look to you.  I'll need your approval before I can merge :-)